contrib/packaging/rpm/cfg/cilium.spec.envsubst

```diff
@@ -0,0 +1,70 @@
+%global commit0 ${COMMIT}
+%global shortcommit0 %(c=%{commit0}; echo ${c:0:7})
+
+Name:          cilium
+Version:       ${VERSION}
+Release:       0.git%{shortcommit0}%{?dist}
+Summary:       BPF & XDP for containers
+License:       Apache
+URL:           https://github/cilium/cilium
+ExclusiveArch: x86_64
+Source0:       https://github.com/cilium/%{name}/archive/%{commit0}.tar.gz#/%{name}-%{shortcommit0}.tar.gz
+BuildRequires: golang < 1.8.0, glibc-devel(x86-32)
+Requires:      docker-engine >= 1.12, clang >= 3.8, glibc-devel(x86-32), iproute >= 4.6, kernel >= 4.8
```

Comment:
I was still able to install it via official repos with `--allowerasing`:
`dnf install clang-3.8.0-2.fc25.x86_64 --allowerasing`
```
$ sudo dnf info clang-3.8.0-2.fc25.x86_64
Last metadata expiration check: 1:06:08 ago on Thu Mar 23 15:54:05 2017.
Installed Packages
Name        : clang
Arch        : x86_64
Epoch       : 0
Version     : 3.8.0
Release     : 2.fc25
Size        : 4.1 M
Repo        : @System
From repo   : fedora
Summary     : A C language family front-end for LLVM
URL         : http://llvm.org
License     : NCSA
Description : clang: noun
            :     1. A loud, resonant, metallic sound.
            :     2. The strident call of a crane or goose.
            :     3. C-language family front-end toolkit.
            : 
            : The goal of the Clang project is to create a new C, C++, Objective C
            : and Objective C++ front-end for the LLVM compiler. Its tools are built
            : as libraries and designed to be loosely-coupled and extensible
```

Do you know how long will they keep it there?